docs/html/reference/pip_install.rst

```diff
@@ -913,8 +913,11 @@ Examples
 
     ::
 
-      $ pip install SomeProject==1.0.4@http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl
-      $ pip install "SomeProject==1.0.4 @ http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl"
+      $ pip install SomeProject@http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl
+      $ pip install "SomeProject @ http://my.package.repo//SomeProject-1.2.3-py33-none-any.whl"
+      $ pip install SomeProject@http://my.package.repo//1.2.3.tar.gz
+      $ pip install SomeProject@git+https://git.repo/some_pkg.git@1.3.1
+      $ pip install SomeProject@git+https://git.repo/some_pkg.git@1.3.1#7921be1537eac1e97bc40179a57f0349c2aee67d
```

Comment:
I think the `#7921be1537eac1e97bc40179a57f0349c2aee67d` part is going to be ignored by pip. So it's probably better not to show it as an example.

Reply:
The last two are not source archive, so they don't match the title. These should probably go in their own section just before "VCS in editable mode".

Reply:
if I understand you right, I should only have ``$ pip install SomeProject@git+https://git.repo/some_pkg.git@1.3.1`` in its own section before "VCS in editable mode" (step 5).  I can call that section "Install a project from VCS".